src/com/twitter/intellij/pants/file/FileChangeTracker.java

```diff
@@ -105,6 +112,30 @@ private static void markDirty(@NotNull VirtualFile file, @NotNull VirtualFileLis
     LOG.debug(String.format("Changed: %s. In project: %s", file.getPath(), inProject));
     if (inProject) {
       markDirty(project);
+      notifyProjectRefreshIfNecessary(file, project);
+    }
+  }
+
+  private static void notifyProjectRefreshIfNecessary(@NotNull VirtualFile file, final Project project) {
+    NotificationListener.Adapter refreshAction = new NotificationListener.Adapter() {
+      @Override
+      protected void hyperlinkActivated(@NotNull Notification notification, @NotNull HyperlinkEvent event) {
+        if (event.getDescription().equals("reimport")) {
```

Comment:
How about flipping this equality check? I don't know that we can be sure that getDescription is not null? I looked at the docs and couldn't tell.